pkg/migration/controllers/migration.go

```diff
@@ -1291,7 +1291,11 @@ func (m *MigrationController) applyResources(
 			return err
 		}
 	}
-
+	// TODO: we should use k8s code generator logic to pluralize
+	// crd resources instead of depending on inflect lib
+	ruleset := inflect.NewDefaultRuleset()
+	ruleset.AddPlural("quota", "quotas")
+	ruleset.AddPlural("prometheus", "prometheuses")
```

Comment:
Shouldn't line 1305 be using this ruleset instead of `inflect.Pluralize`

Reply:
yes, good catch. thanks